app/helpers/job_applications_helper.rb

```diff
@@ -44,10 +47,11 @@ def tab_name(job_application_status)
 
   def tag_status_options(job_application_status)
     case job_application_status
+    when "submitted", "reviewed" then %i[unsuccessful shortlisted interviewing offered]
     when "shortlisted"  then %i[unsuccessful interviewing offered]
-    when "interviewing" then %i[unsuccessful offered]
+    when "interviewing" then %i[unsuccessful_interview offered]
     else
-      %i[unsuccessful shortlisted interviewing offered]
+      []
     end
   end
```

Comment:
If I got the right grasp on these changes, this helper method became the de facto "state machine" for our Job Applications status transitions.

Given the business logic importance of these transitions, could we consider moving this to the `JobApplication` model?  We would also gain the capability of defining model validations based on what `from` -> `to` status changes are allowed or not. That I would rather have! 😅